doc/source/parameters/cfd/physical_properties.rst

```diff
@@ -372,13 +372,13 @@ This model is parameterized using the ``phase change`` subsection
   end
 
 
-* The ``liquidus temperature`` is :math:`T_l`
+* The ``liquidus temperature`` is :math:`T_\text{l}`
```

Comment:
In the previous `.. code-block:: text`, the description for `viscosity liquid` should say `viscosity` instead of `specific heat`

Reply:
I just couldn't write a comment there since it is from a previous PR